r_basics_transform_data/r_basics_transform_data.md

```diff
@@ -20,98 +22,67 @@ After completion of this module, learners will be able to:
 
 @end
 
-@version_history
-1.0.2: revision to correct image links referring to wrong branch
-1.0.3: revamping environment setup section to mirror exactly other R basics modules, including only presenting the content once. Changed the hands-on section to follow appropriate structure. Also replaced hex art alt text with blank tags based on best practices for decorative images.
-1.1.0: update pipe syntax, clarified "follow along" instructions, removed unhelpful image
-1.1.1: Add information about Posit Cloud
-1.1.2: Remove second attribution location
-
-@end 
-
-script:  https://code.jquery.com/jquery-3.6.0.slim.min.js
-
-@gifPreload
-<script>
-(function($) {
+@pre_reqs
 
-  // Get the .gif images from the "data-alt".
-	var getGif = function() {
-		var gif = [];
-		$('img').each(function() {
-			var data = $(this).data('alt');
-			gif.push(data);
-		});
-		return gif;
-	}
-
-	var gif = getGif();
-
-	// Preload all the gif images.
-	var image = [];
+Minimal experience of using the RStudio IDE and writing R code (specifically, within an R Markdown document) is necessary to understand and use this material.  If you can understand and do the following, you'll be able to complete this course:
 
-	$.each(gif, function(index) {
-		image[index]     = new Image();
-		image[index].src = gif[index];
-	});
+* Run a command that's provided to you in the console
+* Use the Environment tab to find a data frame and learn more about it
+* Insert a new code chunk in an R Markdown document
 
-	// Change the image to .gif when clicked and vice versa.
-	$('figure').on('click', function() {
+@end
 
-		var $this   = $(this),
-				$index  = $this.index(),
+r_file: r\_basics\_transform\_data
 
-				$img    = $this.children('img'),
-				$imgSrc = $img.attr('src'),
-				$imgAlt = $img.attr('data-alt'),
-				$imgExt = $imgAlt.split('.');
+good_first_module: true
```

Comment:
Just double checking this field --- I think Elizabeth is intending to use this for automatic generation of "pathways", and this field would be used to determine modules that can appear first in a pathway. We have only a couple modules with this set to true (reproducibility, maybe one more?). Not disagreeing, I think this could be a good way to start a pathway, but I just wanted to flag this in case it was an error.

Reply:
Oh yep I don't think this is a good first module!  That was a mistake!  Good catch!